packages/atomic/src/components/atomic-result/atomic-result.pcss

```diff
@@ -1,5 +1,237 @@
+    'bottom-metadata bottom-metadata bottom-metadata';
+  grid-template-columns: auto 1fr auto;
+  grid-template-rows: auto auto auto auto auto auto;
+}
+
+.result-root.image-none section.visual {
+  display: none;
+}
+
+/* == Density styles == */
+/* Comfortable */
+.result-root.density-comfortable {
+  padding: 2rem 1rem;
+}
+
+.result-root.density-comfortable section.visual {
+  margin-right: 2rem;
+}
+
+.result-root.density-comfortable section.badges {
+  margin-bottom: 1.5rem;
+  height: 2.5em;
+}
+
+.result-root.density-comfortable section.actions {
+  margin-bottom: 1.5rem;
+  height: 2.5em;
+}
+
+.result-root.density-comfortable section.title {
+  font-size: 1.5rem;
+  line-height: 2rem;
+}
+
+.result-root.density-comfortable section.excerpt {
+  font-size: 1rem;
+  line-height: 1.5rem;
+}
+
+.result-root.density-comfortable section.bottom-metadata,
+.result-root.density-comfortable section.excerpt {
+  margin-top: 1.75rem;
+}
+
+/* Normal */
+.result-root.density-normal {
+  padding: 1.5rem 1rem;
+}
+
+.result-root.density-normal section.visual {
+  margin-right: 1.5rem;
+}
+
+.result-root.density-normal section.badges {
+  margin-bottom: 1rem;
+  height: 2rem;
+}
+
+.result-root.density-normal section.actions {
+  margin-bottom: 1rem;
+  height: 2rem;
+}
+
+.result-root.density-normal section.title {
+  font-size: 1.5rem;
+  line-height: 2rem;
+}
+
+.result-root.density-normal section.excerpt {
+  font-size: 0.875rem;
+  line-height: 1.25rem;
+}
+
+.result-root.density-normal section.bottom-metadata,
+.result-root.density-normal section.excerpt {
+  margin-top: 1.25rem;
+}
+
+/* Compact */
+.result-root.density-compact {
+  padding: 1rem;
+}
+
+.result-root.density-compact section.visual {
+  margin-right: 1rem;
+}
+
+.result-root.density-compact section.badges {
+  margin-bottom: 1rem;
+  height: 2rem;
+}
+
+.result-root.density-compact section.actions {
+  margin-bottom: 1rem;
+  height: 2rem;
+}
+
+.result-root.density-compact section.title {
+  font-size: 1.125rem;
+  line-height: 1.5rem;
+}
+
+.result-root.density-compact section.excerpt {
+  font-size: 0.875rem;
+  line-height: 1.25rem;
+}
+
+.result-root.density-compact section.bottom-metadata,
+.result-root.density-compact section.excerpt {
+  margin-top: 1rem;
+}
+
+/* Mixed */
+.result-root.image-large.density-compact section.visual {
+  width: 10.25rem;
+  height: 10.25rem;
+}
+
+/* == Common styles == */
+section.visual {
+  grid-area: visual;
+}
+
+section.badges {
+  grid-area: badges;
+
+  text-align: left;
+}
+
+section.actions {
+  grid-area: actions;
+
+  text-align: right;
+}
+
+section.title {
+  grid-area: title;
+
+  color: var(--atomic-on-background);
+}
+
+section.title-metadata {
+  grid-area: title-metadata;
+
+  font-size: 0.75rem;
+}
+
+section.emphasized {
+  grid-area: emphasized;
+
+  font-weight: 500;
+  font-size: 1.5rem;
+  line-height: 2rem;
+  margin-top: 0.5rem;
+}
+
+section.excerpt {
+  grid-area: excerpt;
+
+  color: #626971;
```

Comment:
I'm not sure what to do about this color, should I just use `atomic-neutral` or `atomic-dark`? Will those change when we use a dark theme?

Reply:
There's no atomic-dark.

Dark theme... you can make any theme using the current CSS variables

Reply:
woops I meant `atomic-neutral-dark`.

Makes sense